tools/sdk/include/user_interface.h

```diff
@@ -427,6 +428,17 @@ typedef enum {
     MODEM_SLEEP_T
 } sleep_type_t;
 
+typedef enum {
+    MIN_SLEEP_T,
+    MAX_SLEEP_T
+} sleep_level_t;
```

Comment:
Is there info about what this sleep level means?

Reply:
Question [already asked](https://github.com/espressif/ESP8266_NONOS_SDK/commit/236b6d0adb80eb52ab4f5d86a62cdb4d58f0daae#commitcomment-28967677), no answer yet. Reading actual values and playing with them may help #2330.